### PR TITLE
[#9321] Release 7.0.0-alpha.0

### PR DESCRIPTION
**Pull request by wkurniawan07**
Part of #9321

This is not the usual release PR because some touch ups are done here:
- Removal of items that are already migrated
- Restructure of new controller packages (third commit)

Along with the usual `developers.json` update.

### CODE REVIEW

File: static-analysis/teammates-macker.xml [hunk 1]
```diff
@@ -78,21 +78,47 @@
...
                             </from>
                             <to class="${logic}.api.**" />
                         </allow>
                     </deny>
                 </access-rule>
             </ruleset>
+            <ruleset name="UiComponent: API output is only used in Web API package">
+                <access-rule>
+                    <deny>
+                        <from class="${ui}.**" >
```
Comment (xpdavid) [inline static-analysis/teammates-macker.xml]: This `from` is redundant?

Comment (wkurniawan07) [reply static-analysis/teammates-macker.xml]: Don't think so: it blocks the `ui.automated` package from accessing.

Comment (xpdavid) [reply static-analysis/teammates-macker.xml]: `ui.automated` needs access to API output?

File: src/main/java/teammates/common/util/Const.java [hunk 2]
```diff
@@ -1229,22 +1118,12 @@ private Const() {
         public static final String INSTRUCTOR_FEEDBACK_RESULTS_BY_RECIPIENT_QUESTION_GIVER =
                 "/jsp/instructorFeedbackResultsByRecipientQuestionGiver.jsp";
         public static final String INSTRUCTOR_FEEDBACK_RESULTS_BY_QUESTION = "/jsp/instructorFeedbackResultsByQuestion.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT = "/jsp/instructorFeedbackSubmissionEdit.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_QUESTION_SUBMISSION_EDIT =
-                "/jsp/instructorFeedbackQuestionSubmissionEdit.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_QUESTION_COPY_MODAL =
```
[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/main/java/teammates/common/util/Const.java]: Let's use `release` branch copy as reference.

File: src/main/java/teammates/common/util/Const.java [hunk 3]
```diff
@@ -1188,36 +1098,15 @@ private Const() {
...
                 "/jsp/instructorFeedbackResponseCommentsAdd.jsp";
-        public static final String INSTRUCTOR_COURSE_EDIT = "/jsp/instructorCourseEdit.jsp";
-        public static final String INSTRUCTOR_COURSE_DETAILS = "/jsp/instructorCourseDetails.jsp";
-        public static final String INSTRUCTOR_COURSE_ENROLL = "/jsp/instructorCourseEnroll.jsp";
-        public static final String INSTRUCTOR_COURSE_ENROLL_RESULT = "/jsp/instructorCourseEnrollResult.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_SESSIONS = "/jsp/instructorFeedbacks.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_COPY_MODAL = "/jsp/instructorFeedbackCopyModal.jsp";
         public static final String INSTRUCTOR_FEEDBACK_AJAX_REMIND_PARTICULAR_STUDENTS_MODAL =
                 "/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp";
-        public static final String INSTRUCTOR_FEEDBACK_AJAX_RESEND_PUBLISHED_EMAIL_MODAL =
```
[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/main/java/teammates/common/util/Const.java]: Let's use `release` branch copy as reference.

File: static-analysis/teammates-macker.xml (hunk 1 above)
Comment (wkurniawan07) [reply static-analysis/teammates-macker.xml]: No, it does not. Although technically speaking `ui.automated` is also a RESTful API package, we don't require them to have properly defined output at least for now.

**Pull request merged**